platform-includes/debug-symbols-apple/_default.mdx

```diff
@@ -80,6 +80,7 @@ Follow these steps to upload the debug symbols:
 
 1.  Download and install [sentry-cli](/product/cli/installation/).
 2.  Copy the script below into a new _Run Script_ and set your _AUTH_TOKEN_, _ORG_SLUG_, and _PROJECT_SLUG_.
+3.  Set `ENABLE_USER_SCRIPT_SANDBOXING` in your Xcode project settings to `NO`. <img src="enable_user_script_sandboxing.png"/>
```

Comment:
```suggestion
3.  Set `ENABLE_USER_SCRIPT_SANDBOXING` in your Xcode project settings to `NO`. 
  [enable-user-script-sandboxing](./img/enable_user_script_sandboxing.png
```
You need to use markdown formatting and move the image to a `img/` folder in the same directory (platform-includes/debug-symbols-apple/) as the doc. See [this page](https://docs.sentry.io/contributing/pages/images/) for more details